src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
this file can be deleted since you will be removing most of the content from the App.js file! You can recreate it later if you end up needing it